.github/actions/generate-coverage/tests/test_scripts.py

```
@@ -0,0 +1,116 @@
+        "RUST_FILE": str(rust),
+        "PYTHON_FILE": str(py),
+        "OUTPUT_PATH": str(out),
+    }
+    script = Path(__file__).resolve().parents[1] / "scripts" / "merge_cobertura.py"
+    res = run_script(script, env)
+    assert res.returncode == 0
+    assert out.read_text() == "<merged/>"
+    assert not rust.exists()
+    assert not py.exists()
+    calls = shell_stubs.calls_of("uvx")
+    assert calls and calls[0].argv[:1] == ["merge-cobertura"]
```

Ruff (0.11.9)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)